crates/rpc/rpc/src/trace.rs

```diff
@@ -239,14 +239,21 @@ where
         filter: TraceFilter,
     ) -> EthResult<Vec<LocalizedTransactionTrace>> {
         let matcher = filter.matcher();
-        let TraceFilter { from_block, to_block, after: _after, count: _count, .. } = filter;
+        let TraceFilter { from_block, to_block, .. } = filter;
         let start = from_block.unwrap_or(0);
         let end = if let Some(to_block) = to_block {
             to_block
         } else {
             self.provider().best_block_number()?
         };
 
+        if end > start {
```

Comment:
it should be the other way around I believe?

```suggestion
        if start > end {
```

Reply:
Sorry for the careless :)